static/js/components/search/CustomNoHits.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import React from 'react';
+import { NoHits } from 'searchkit';
+
+export default class CustomNoHits extends NoHits {
+  render() {
+    if ((this.hasHits() || this.isInitialLoading() || this.isLoading()) && !this.getError()) {
```

Comment:
That's a little concerning. What happens if `this.getError()` returns errors? This `render()` function will return `null`, and the user will just see a blank screen instead of the error messages.

Can you add a test where `this.hasHits()` returns `false`, but `this.getError()` includes at least one error? I would expect the screen to show that error message to the user, so the user knows *why* they're having a problem.